source/renderer/app/containers/voting/dialogs/VotingRegistrationDialogContainer.tsx

```diff
@@ -269,30 +273,32 @@ class VotingRegistrationDialogContainer extends Component<Props, State> {
       const selectedWallet = getWalletById(this.selectedWalletId);
       const [address] = await getAddressesByWalletId(this.selectedWalletId);
       const isHardwareWallet = get(selectedWallet, 'isHardwareWallet', false);
-      let fee;
+      let coinSelection;
       let votingData;
 
       if (isHardwareWallet) {
         votingData = await prepareVotingData({
           walletId: this.selectedWalletId,
         });
-        ({ fee } = await selectCoins({
+        coinSelection = await selectCoins({
           walletId: this.selectedWalletId,
           address: address.id,
           amount,
           metadata: votingData.metadata,
-        }));
+        });
+
+        updateTxSignRequest(coinSelection);
       } else {
-        ({ fee } = await calculateTransactionFee({
+        coinSelection = await calculateTransactionFee({
```

Comment:
i don't think `coinSelection` is the appropriate name in this context.  TransactionStore.calculateTransactionFee returns a different data structure.

Reply:
Good catch, I got confused by destructuring which looked like we're dealing with the same model. Changed now.